src/shared/components/secrets/CreateSecretOverlay.tsx

```diff
@@ -0,0 +1,35 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+
+// Components
+import {Overlay} from '@influxdata/clockface'
+import CreateSecretForm from 'src/shared/components/secrets/CreateSecretForm'
+
+// Contexts
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+// Types
+import ErrorBoundary from 'src/shared/components/ErrorBoundary'
+import GetResources from 'src/resources/components/GetResources'
+import {ResourceType} from 'src/types'
+
+const CreateSecretOverlay: FC = () => {
+  const {onClose} = useContext(OverlayContext)
+
+  return (
+    <Overlay.Container maxWidth={750}>
+      <Overlay.Header
+        title="Add Secret"
+        onDismiss={onClose}
+        testID="create-secret-overlay-header"
+      />
+      <GetResources resources={[ResourceType.Secrets]}>
```

Comment:
rough

Reply:
good catch. copy/pasta of old component (a.k.a. slop).
is not used in the child component (which has the newer approach to dispatching).